server.js

```diff
@@ -3768,23 +3767,38 @@ function mapNugetFeed(pattern, offset, getInfo) {
 
           try {
             var data = JSON.parse(buffer);
-            var versions = data.data;
-            if (!includePre) {
-              // Remove prerelease versions.
-              var filteredVersions = versions.filter(function(version) {
-                return !/-/.test(version);
-              });
-              if (filteredVersions.length > 0) {
-                versions = filteredVersions;
-              }
+            data = data.data && data.data[0];
```

Comment:
Could you make this a more explicit check that it's an array, and with the expected length?